contrib/download-frozen-image-v2.sh

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+# This script was developed for use in Moby's CI, and as such the use cases and
```

Comment:
Nit: this paragraph may appear before set -eo pipefail

Reply:
oh! TBH, I wasn't 100% sure if that would mess up things, so I took the "safe" approach ☺️